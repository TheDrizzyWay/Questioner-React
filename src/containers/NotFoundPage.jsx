import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
    return (
        <div id="notfound">
            <div className="notfound-bg" />
            <div className="notfound">
                <div className="notfound-404">
                    <h1>404</h1>
                </div>
                <h2>So many things exist but not this page.</h2>
                <Link to="/" className="home-btn">Go Home</Link>
            </div>
        </div>
    );
};

export default NotFoundPage;
