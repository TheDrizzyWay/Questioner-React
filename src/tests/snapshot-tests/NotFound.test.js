import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NotFoundPage from '../../containers/LandingPage';

describe('NotFoundPage', () => {
    test('Should maintain existing snapshot', () => {
        const component = renderer.create(
            <Router><NotFoundPage /></Router>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
