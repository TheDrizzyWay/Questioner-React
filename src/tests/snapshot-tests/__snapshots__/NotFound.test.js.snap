// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Should maintain existing snapshot 1`] = `
Array [
  <div
    className="header_container"
  >
    <header>
      <div
        className="header_div"
      >
        <h1>
          Questioner
        </h1>
        <div
          className="details"
        >
          <p>
            Schedule and organize meetups 
            <br />
            with no fuss.
          </p>
          <a
            className="link-style"
            href="/login"
            onClick={[Function]}
          >
            <button
              className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedPrimary-13 MuiButton-raised-15 MuiButton-raisedPrimary-16"
              disabled={false}
              name="button"
              onBlur={[Function]}
              onContextMenu={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex="0"
              type="button"
            >
              <span
                className="MuiButton-label-2"
              >
                Get Started
              </span>
              <span
                className="MuiTouchRipple-root-30"
              />
            </button>
          </a>
        </div>
      </div>
    </header>
  </div>,
  <footer>
    <p>
      Powered by TheDrizzyWay Â© 2018
    </p>
    <br />
  </footer>,
]
`;
